### PR TITLE
[TW-83370] Update .NET SDKs according to LTS / STS versions supported by Microsoft.

### PR DESCRIPTION
**Pull request by AndreyKoltsov1997**
The .NET SDK version bundled within TeamCity Docker Images will be aligned with [Microsoft's Long Term Support (LTS) release](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) at the moment of TeamCity release. Given the potential necessity to use STS (which is sometimes newer than LTS) or 
older versions, pull request includes the examples of building images with custom .NET SDK versions inside.

### CODE REVIEW

File: configs/windows/Agent/windowsservercore/WindowsServerCore1803.Dockerfile [hunk 1]
```diff
@@ -59,17 +49,14 @@ ARG mercurialWindowsComponent
...
-# .NET 3.1
-    Expand-Archive dotnet_31.zip -Force -DestinationPath $Env:ProgramFiles\dotnet; \
-    Remove-Item -Force dotnet_31.zip; \
-# .NET 5.0
-    Expand-Archive dotnet_50.zip -Force -DestinationPath $Env:ProgramFiles\dotnet; \
-    Remove-Item -Force dotnet_50.zip; \
 # .NET 6.0
     Expand-Archive dotnet.zip -Force -DestinationPath $Env:ProgramFiles\dotnet; \
     Remove-Item -Force dotnet.zip; \
+# .NET 7.0
```
Comment (burnasheva) [inline configs/windows/Agent/windowsservercore/WindowsServerCore1803.Dockerfile:57]: Why do we have .NET 7.0?

File: custom/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Custom TeamCity Agent Images
```
Comment (burnasheva) [inline custom/README.md:1]: @Valrravn Could you please wording here?

**Review by burnasheva — CHANGES_REQUESTED** [review]

File: configs/windows/Agent/windowsservercore/WindowsServerCore1803.Dockerfile (hunk 1 above)
Comment (AndreyKoltsov1997) [reply configs/windows/Agent/windowsservercore/WindowsServerCore1803.Dockerfile]: Great catch, thank you, @burnasheva !
It was mistakenly added back via the reset of some of the experimental changes. Fixed by the commit https://github.com/JetBrains/teamcity-docker-images/pull/111/commits/6eb2d7f9d45f2a7897d3a07454b5e6eceb174d4b

File: custom/README.md (hunk 2 above)
Comment (Valrravn) [reply custom/README.md:1]: edited the entire readme a bit

**Pull request merged**